### DISCUSSION TITLE
Ubuntu 20.4 installation

### DISCUSSION THREAD

**Discussion by h1dd3nsn1p3r** [dominiksalvet/asus-fan-control#110] [Q&A] [answered]
Hi guys,

It looks like this project is made for Arch Linux. I am using Ubuntu 20.4 (Zorin OS 16 UI). 

I would like to know if this works for ubuntu OS? I cloned the repo but, I got `sudo: makepkg: command not found` while trying to run it. 

My PC is Asus Vivobook S15. Yes, it is not listed as tested but I would like to test it myself. 

Thank you

**Comment by dominiksalvet** [comment, marked as answer]
Hello @h1dd3nsn1p3r. I use Ubuntu 20.04, and actually it was the first OS tested. However, you just must use GitPack for installation (it is universal) instead of the AUR method.

**Reply by h1dd3nsn1p3r** [reply]
Hi @dominiksalvet Thank you very much. It looks like I had overlooked GitPack. Now I have successfully installed Asus Fan Control. I'll let you know the feedback soon. 

I have also added `"sudo asus-fan-control"` to the startup.
